### DISCUSSION TITLE
code micon pour dacia version 9.0.12.2

### DISCUSSION THREAD

**Discussion by titus091994** [cedricp/ddt4all#1301] [Q&A]
Bonjour

je suis a la recherche des codes micon pour dacia duster 2023 MERCI

code micon pour dacia version 9.0.12.2

**Comment by Furtif** [comment]
Désole, ceci n'est pas un issue (problème), je redirectionne votre question dans [Discussions](https://github.com/cedricp/ddt4all/discussions)

J'ai demander a Jeremy peut être il a la réponse a votre problème, toute fois vous pouvais interagit directement dans leur Group [facebook](https://www.facebook.com/groups/140606960090892)  

Espérons qu'il ai la solution.

**Comment by titus091994** [comment]
Bonjour recu cable remolink 1.99 que faut il faire pour mettre android auto en couleur merci?

**Reply by matzel82** [reply]
Renolink. I own Renault-Link too. But is better to use DDT. Or you can ask at Renolink Support/Developer. Where you buy Renolink ?

**Comment by titus091994** [comment]
https://obd-diag.fr/produit/renolink-programmateur-ecu-v1-94-v1-87-cable-de-diagnostic-obd2-interface-pour-renault-dacia-codage-de-cle-airbag-reno-link-1-87/

**Reply by titus091994** [reply]
il me faudrait la modif du cable 2 males 1 femelle pour le can 2 pour dacia duster

**Reply by titus091994** [reply]
pour modif radio

**Reply by titus091994** [reply]
merci
